test/Shared/UserManagerTestBase.cs

```diff
@@ -568,17 +573,16 @@ protected TRole CreateRole(string namePrefix = "")
             IdentityResultAssert.IsSuccess(await manager.AddLoginAsync(user, login));
             var result = await manager.AddLoginAsync(user, login);
             IdentityResultAssert.IsFailure(result, IdentityErrorDescriber.Default.LoginAlreadyAssociated());
-            IdentityResultAssert.VerifyUserManagerFailureLog(manager.Logger, "AddLoginAsync", user.Id.ToString(), IdentityErrorDescriber.Default.LoginAlreadyAssociated());
+            IdentityResultAssert.VerifyUserManagerFailureLog(manager.Logger, "AddLoginAsync", await manager.GetUserIdAsync(user), IdentityErrorDescriber.Default.LoginAlreadyAssociated());
         }
 
         // Email tests
         [Fact]
         public async Task CanFindByEmail()
         {
+            var email = "foouser@test.com";
```

Comment:
The requireuniqueemail flag is false so this is passing with all tests. We can have a random number with the email and set that flag to true